projects/demo-playwright/tests/addon-charts/line-chart/line-chart.spec.ts

```diff
@@ -0,0 +1,15 @@
+import {TuiDocumentationPagePO, tuiGoto} from '@demo-playwright/utils';
+import {expect, test} from '@playwright/test';
+
+test.describe(`LineChart`, () => {
+    test(`should not show hint`, async ({page}) => {
+        await tuiGoto(page, `charts/line-chart`);
+        const example = new TuiDocumentationPagePO(page).getExample(`#line`);
+        const lineChart = example
```

Comment:
Actually, this variable keeps locator not for the whole chart but the part of it.
Should we rename variable ? 🤔 

```suggestion
        const chartColumn = example
```

Reply:
Makes sense. Sorry for this!